package.json

```diff
@@ -57,7 +57,7 @@
     "@types/glob": "^5.0.33",
     "@types/gulp": "3.8.32",
     "@types/hammerjs": "^2.0.35",
-    "@types/jasmine": "^2.6.0",
+    "@types/jasmine": "2.2.34",
```

Comment:
@jelbourn I've bumped this one down temporarily due to a compilation error coming in from core. I've submitted https://github.com/angular/angular/pull/23110 to fix it.